plugins/source/aws/views/resources.sql

```diff
@@ -14,14 +14,17 @@ FOR tbl IN
     SELECT table_name
     FROM information_schema.columns
     WHERE table_name LIKE 'aws_%s' and COLUMN_NAME = 'arn'
-LOOP 
+LOOP
     -- UNION each table query to create one view
  	IF NOT (strSQL = ''::TEXT) THEN
 		strSQL = strSQL || ' UNION ALL ';
 	END IF;
 	-- create an SQL query to select from table and transform it into our resources view schema
-	strSQL = strSQL || FORMAT('
-        select _cq_id, _cq_source_name, _cq_sync_time, %L as _cq_table, account_id, %s as region, arn, %s as tags
+	strSQL = strSQL || FORMAT(E'
+        SELECT _cq_id, _cq_source_name, _cq_sync_time, %L AS _cq_table, account_id, %s AS region,
+        SPLIT_PART(arn, \':\', 2) AS PARTITION, SPLIT_PART(arn, \':\', 3) AS service,
+        CASE WHEN SPLIT_PART(SPLIT_PART(arn, \':\', 6), \'/\', 2) = \'\' AND SPLIT_PART(arn, \':\', 7) = \'\' THEN NULL ELSE SPLIT_PART(SPLIT_PART(arn, \':\', 6), \'/\', 1) END AS type,
+        arn, %s AS tags
         FROM %s',
```

Comment:
Can we make this a little easier to read like this?
```suggestion
        SELECT _cq_id, 
        	_cq_source_name, 
        	_cq_sync_time, 
        	%L AS _cq_table, 
        	account_id, 
        	%s AS region,
        	SPLIT_PART(arn, \':\', 2) AS PARTITION, 
        	SPLIT_PART(arn, \':\', 3) AS service,
        	CASE
				WHEN SPLIT_PART(SPLIT_PART(ARN, \':\', 6), \'/\', 2) = \'\' AND SPLIT_PART(arn, \':\', 7) = \'\' THEN NULL
        		ELSE SPLIT_PART(SPLIT_PART(arn, \':\', 6), \'/\', 1)
        	END AS TYPE,
        	arn, %s AS tags
        FROM %s',
```